### PR TITLE
VTCCA-3728: is viewable external fix

### PR DESCRIPTION
**Pull request by darrell-plant**
Remove flag and code (feature.externalValuation)
Fix due to change handling of 404s in http-verbs library (when calling 'is detailed valuation viewable')
Small refactoring

### CODE REVIEW

File: app/uk/gov/hmrc/propertylinking/errorhandler/CustomErrorHandler.scala [hunk 1]
```diff
@@ -58,8 +58,9 @@ class CustomErrorHandler @Inject()()(implicit override val messagesApi: Messages
       case error: BraAuthorisationFailure =>
         logger.info(s"business rates authorisation returned ${error.message}, redirecting to login.")
         Future.successful(Redirect(appConfig.ggSignInUrl, Map("continue" -> Seq(request.uri), "origin" -> Seq("voa"))))
-      case _ =>
+      case _ => {
+        Logger.warn(s"********************** onServerError")
```
Comment (PeterPerhac) [inline app/uk/gov/hmrc/propertylinking/errorhandler/CustomErrorHandler.scala:62]: did you intend to keep this?

**Comment by PeterPerhac** [comment]
👍 looks good, but I think that one warn log should be removed

File: app/uk/gov/hmrc/propertylinking/errorhandler/CustomErrorHandler.scala (hunk 1 above)
Comment (darrell-plant) [reply app/uk/gov/hmrc/propertylinking/errorhandler/CustomErrorHandler.scala:62]: ha! thought I'd removed all
Thanks

**Comment by habeeb01** [comment]
👍  LGTM

**Pull request merged**